templates/parachain/runtime/src/lib.rs

```diff
@@ -489,6 +489,7 @@ impl pallet_collator_selection::Config for Runtime {
 /// Configure the pallet template in pallets/template.
 impl pallet_parachain_template::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = pallet_parachain_template::weights::SubstrateWeight<Runtime>;
```

Comment:
Maybe put a comment here since this is using the Substrate default instead of custom benchmarked weights?  
I saw production parachains doing as well a few times already...

Reply:
I still hope we'd had something like https://github.com/paritytech/polkadot-sdk/issues/218 and then we could ban the usage of these `SubstrateWeights` in production. 

We can still hack around it by just making them emit a warning in `no_std`.

Reply:
>We can still hack around it by just making them emit a warning in no_std

Yea good idea actually.